pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -226,3 +226,12 @@ type TriggerList struct {
 func (t *Trigger) GetStatus() *duckv1.Status {
 	return &t.Status.Status
 }
+
+func (t *Trigger) GetCrossNamespaceRef() duckv1.KReference {
+	return duckv1.KReference{
+		APIVersion: t.APIVersion,
+		Kind:       t.Kind,
+		Name:       t.Name,
+		Namespace:  t.Namespace,
```

Comment:
Shouldn't this be the `brokerRef` value on the trigger, since this is used to check the cross namespace target?